bpf/process/types/kprobes.h

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Tetragon */
+
+#ifndef __LINUX_KPROBES_H__
+#define __LINUX_KPROBES_H__
+
+#ifndef KSYM_NAME_LEN
+#define KSYM_NAME_LEN 128U
+#endif
+
+struct msg_kprobe {
```

Comment:
We also may want to capture `flags` here, as it contains some useful [state info](https://github.com/torvalds/linux/blob/e8f897f4afef0031fe618a8e94127a0934896aba/include/linux/kprobes.h#L96):

```c
/* Kprobe status flags */
#define KPROBE_FLAG_GONE	1 /* breakpoint has already gone */
#define KPROBE_FLAG_DISABLED	2 /* probe is temporarily disabled */
#define KPROBE_FLAG_OPTIMIZED	4 /*
				   * probe is really optimized.
				   * NOTE:
				   * this flag is only for optimized_kprobe.
				   */
#define KPROBE_FLAG_FTRACE	8 /* probe is using ftrace */
#define KPROBE_FLAG_ON_FUNC_ENTRY	16 /* probe is on the function entry */
```